hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeTest.java

```diff
@@ -145,5 +145,23 @@ public void testMapCount() {
         parameters = phoneHome.phoneHome(true);
         assertEquals(parameters.get("mpct"), "3");
     }
+
+    @Test
+    public void testMapCountWithBackupReadEnabled() {
+        HazelcastInstance hz = createHazelcastInstance();
+        Node node = getNode(hz);
+        PhoneHome phoneHome = new PhoneHome(node);
```

Comment:
These 3 lines seem to be duplicated in all unittests. Can you extract it into a `@Before` method and have the `node` and `phoneHome` initialized as fields?